### DISCUSSION TITLE
connectUnchecked is unsupported

### DISCUSSION THREAD

**Discussion by desenvolvedorblockchain** [wevm/wagmi#1279] [Q&A]
Hello everyone,

We are trying to create a simple Mint page with Sequence + wagmi + infuraProvider using this project [https://github.com/0xsequence/demo-dapp-rainbowkit](url), and right now we are facing this error:

**mutation.mjs:134 Error: connectUnchecked is unsupported
    at Web3Signer.connectUnchecked (0xsequence-provider.esm.js:1402:1)
    at sendTransaction (wagmi-core.esm.js:1678:1)
    at async writeContract (wagmi-core.esm.js:1770:1)**

We have this code:

```
const provider = useProvider()
const { data: signer } = useSigner()

const { address } = useAccount();

let abiCoder = ethers.utils.defaultAbiCoder;
let dataEncoded = abiCoder.encode(["uint[]"], [[]]); 

const { config } = usePrepareContractWrite({
    address: '0x000', // Contract address
    abi: ERC_1155_ABI, // basic ERC1155 contract
    functionName: 'mint',
    args: [
      address, // address to mint
      0, // ids
      1, // amount
      dataEncoded,
    ],
    overrides: {
      value: ethers.utils.parseEther('0.01'),
    },
})
const { data, error, isError, write } = useContractWrite(config);
```
  
and then, in the button, we used:
  
```
<Button disabled={!write} onClick={() => write?.()}>
      Mint NFT
</Button>
```

to call the write from useContractWrite

We searched all over the internet to find out what this error: "connectUnchecked is unsupported" could be, but we didn't find anything.

Thank you!

**Comment by tmm** [comment]
Mind formatting the code above? It's hard to follow. If you post a [minimal reproducible example](
https://stackoverflow.com/help/minimal-reproducible-example), I'm happy to take a look.

**Reply by desenvolvedorblockchain** [reply]
made it, it was not working in the first time, thank you!
